### DISCUSSION TITLE
Roadmap v1

### DISCUSSION THREAD

**Discussion by Dev4w4n** [Dev4w4n/e-masjid.my#26] [Ideas]
cancelled

**Comment by jebatco** [comment]
![e-masjid](https://github.com/Dev4w4n/e-masjid.my/assets/6611845/3868d833-3784-4008-99e6-1f715bcc4593)

**Comment by jebatco** [comment]
![image](https://github.com/user-attachments/assets/79cf5146-5c2b-4236-90ba-9872e2f7ee77)

**Roadmap.v3**
